modules/host/contractmanager/consts.go

```diff
@@ -76,6 +71,13 @@ var (
 		Header:  "Sia Contract Manager WAL",
 		Version: "1.2.0",
 	}
+
+	// maxWalSize defines the maximum size in bytes of the WAL
+	maxWalSize = build.Select(build.Var{
+		Dev:      int64(5e+5),
+		Standard: int64(5e+6),
+		Testing:  int64(5e+4),
```

Comment:
for readability, we only use powers of 3 in our exponentiation. That makes match the phonetics (50e3 = 50 thousand :: 5e4 = 5 ten-thousands?). In my experience, it's much easier to read. So you'd have 50e3, 500e3, and 5e6 for your values above.